staging/src/k8s.io/cli-runtime/pkg/resource/builder.go

```diff
@@ -276,6 +282,17 @@ func (b *Builder) FilenameParam(enforceNamespace bool, filenameOptions *Filename
 	return b
 }
 
+// expandIfFilePattern returns all the filenames that match the input pattern
+// or the filename if it is a specific filename and not a pattern.
+// If the input is a pattern and it yields no result it will result into an error.
+func (b *Builder) expandIfFilePattern(pattern string) ([]string, error) {
```

Comment:
`Builder` is on `cli-runtime`, are we sure we want to reference `kubectl` from it and create a cycle? We can create another util helper file if we want to remove this responsibility from `Builder`.

Reply:
@danielrodriguez , yea you are right, my apology! what about just 
```
func expandIfFilePattern(pattern string)
```
since we are not using the Receiver?